### PR TITLE
[ChargeItem] [R4] Add 'replacing' custom extension 

### PR DESCRIPTION
**Pull request by sujanb12**
Description
----
This pr adds a new custom extension 'replacing' to the structure definition and charge Item structure definition list
Screen Shot 2021-04-22 at 9 44 10 AM
Screen Shot 2021-04-22 at 10 19 31 AM
Screen Shot 2021-04-22 at 10 12 56 AM
Screen Shot 2021-04-22 at 11 56 32 AM

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Review by ahamdany — APPROVED** [review]

File: content/millennium/r4/financial/charge-item.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ The following fields are returned if valued:
 * [Performing Location]
 * [Procedure]
 * [Revenue Code]
+* [Replacing]
```
Comment (exopsRamba) [inline content/millennium/r4/financial/charge-item.md:48]: nit: `Replacing` should be before revenue code

File: content/millennium/r4/conformance/structure-definition.md [hunk 2]
```diff
@@ -106,6 +106,7 @@ Authorization is not required.
  [`related-person-encounter`]                                | Reference to the Encounter associated to the encounter level RelatedPerson.
  [`relationship-level`]                                      | The resource's relationship to either the patient or encounter level.
  [`replaced-by`]                                             | The resource containing this link must no longer be used. The link points forward to another resource that must be used in lieu of the resource that contains this link.
+ [`replacing`]                                                | A reference to a resource that this resource is replacing.
```
Comment (exopsRamba) [inline content/millennium/r4/conformance/structure-definition.md:109]: This is spaced over once more than everything else.

Comment (sujanb12) [reply content/millennium/r4/conformance/structure-definition.md]: updated [here](https://github.com/cerner/fhir.cerner.com/pull/626/commits/c8e893c718e3d46fdf3d29cf2ab1e5107f7dc9e1#diff-f10dc1eb3735aa1264e82ae17b8a0cd967fc69912568786cd6406be319d3fd17R109)

**Review by Coffeevee — APPROVED** [review]

File: content/millennium/r4/financial/charge-item.md (hunk 1 above)
Comment (sujanb12) [reply content/millennium/r4/financial/charge-item.md:47]: updated [here](https://github.com/cerner/fhir.cerner.com/pull/626/commits/9db66e25714bc87e3463b174c9adb8e5506567d0#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR48)

File: content/millennium/r4/financial/charge-item.md [hunk 3]
```diff
@@ -44,6 +44,7 @@ The following fields are returned if valued:
 * [Offset By]
 * [Performing Location]
 * [Procedure]
+* [Replacing]
```
Comment (alexbeckner1) [inline content/millennium/r4/financial/charge-item.md:47]: I think we may need Replacing listing in the Extensions section of "The following fields are returned if valued:" in the Overview

Comment (sujanb12) [reply content/millennium/r4/financial/charge-item.md:47]: Added [here](https://github.com/cerner/fhir.cerner.com/pull/626/commits/a8d4a0280e037886cfdcd8149eb3eb1e4abceb6a#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR32).
Screenshot updated above.

Comment (alexbeckner1) [reply content/millennium/r4/financial/charge-item.md:47]: Thanks

**Review by alexbeckner1 — APPROVED** [review]

**Review by sydneanschutz — APPROVED** [review]

**Comment by mayur-rajendran** [comment]
Published Verification:

Screen Shot 2021-04-23 at 12 34 16 PM

Screen Shot 2021-04-23 at 12 33 04 PM

Screen Shot 2021-04-23 at 12 33 25 PM

**Pull request merged**